build.zig

```diff
@@ -127,6 +119,48 @@ pub fn build(b: *std.Build) !void {
         .root_source_file = b.path("src/vsr.zig"),
     });
     vsr_module.addImport("vsr_options", vsr_options_module);
+    switch (tracer_backend) {
+        .none => {},
+        .tracy => {
+            // Code here is based on
+            // https://github.com/ziglang/zig/blob/a660df4900520c505a0865707552dcc777f4b791/build.zig#L382
+
+            // On mingw, we need to opt into windows 7+ to get some features required by tracy.
+            const tracy_c_flags: []const []const u8 = if (target.result.os.tag == .windows and target.result.abi == .gnu)
+                &[_][]const u8{
+                    "-DTRACY_ENABLE=1",
+                    "-DTRACY_FIBERS=1",
+                    "-fno-sanitize=undefined",
+                    "-D_WIN32_WINNT=0x601",
+                }
+            else
+                &[_][]const u8{
+                    "-DTRACY_ENABLE=1",
+                    "-DTRACY_FIBERS=1",
+                    "-fno-sanitize=undefined",
+                };
+
+            const tracy = b.addSystemCommand(&.{
```

Comment:
neat

Reply:
I have a slight suspicion that something like this was actually possible back in 0.9 days, and that my shell.zig based GitCloneStep was a pile of neeedless accidental complexity from the start, but I didn't pursue that inquiry in order to remain optimistic :)